clients/privacy-center/components/modals/consent-request-modal/ConsentRequestForm.tsx

```diff
@@ -188,15 +191,13 @@ const ConsentRequestForm: React.FC<ConsentRequestFormProps> = ({
             </Text>
           ) : null}
           <Stack spacing={3}>
-            {config.consent?.identity_inputs.email ? (
+            {identityInputs.email ? (
               <FormControl
                 id="email"
                 isInvalid={touched.email && Boolean(errors.email)}
               >
                 <FormLabel>
-                  {config.consent?.identity_inputs.email === "required"
-                    ? "Email*"
-                    : "Email"}
+                  {identityInputs.email === "required" ? "Email*" : "Email"}
```

Comment:
This is likely out of scope for this PR, but why not use the `isRequired` property on the `FormControl` to add the asterisk? https://chakra-ui.com/docs/components/form-control#making-a-field-required

Reply:
If we should be changing it, then it might be worth just doing it as a followup, since it's probably got a larger reach.

Reply:
yeah, that's probably a good idea! I agree it's out of the scope of this PR though, but I'll make a follow up issue

Reply:
https://github.com/ethyca/fides/issues/2269